### PR TITLE
added home button and styles menu for hover and active state

### PR DESCRIPTION
**Pull request by katieolson84**
## Description

I added a button for the Home page, Paul will change the route to this component, but it has been changed to "/" and I reference a new route of "/cases" that will need to be built. I also updated the hover and active states of the buttons. 

Known Bugs:
- The nav does not expand to the end of the page in the components that have tables. The tables are exceeding the frame and will need to be adjusted in styling. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by MichelleCCodes — APPROVED** [review]
The continued application of global themes keeps styling consistent, keep it up Katie!

File: src/components/pages/SideDrawer/SideDrawer.js [hunk 1]
```diff
@@ -46,7 +46,11 @@ function SideDrawer() {
           )}
         </Button>
 
-        <Menu.Item className="cases" key="/">
+        <Menu.Item className="home" key="/">
```
Comment (paulstgermain) [inline src/components/pages/SideDrawer/SideDrawer.js:49]: This is great - I've already built the routing out for the new component, so this should match perfectly when I merge that code in later today!

File: src/components/pages/SideDrawer/_SideDrawer.less [hunk 2]
```diff
@@ -86,21 +131,28 @@ li {
...
 }
 
+// l.drop-down-item {
+//   padding-right: 0 !important;
+// }
+
 .ant-menu-item-selected {
-  color: @selected-color;
+  color: @gray;
+  background-color: @selected-color !important;
```
Comment (paulstgermain) [inline src/components/pages/SideDrawer/_SideDrawer.less:140]: Looks like you figured out how to get active status styling working on the menu - that's great!

**Review by paulstgermain — APPROVED** [review]
Looks like the usual clean, effective code - approving these changes as an excellent iteration of the sidebar and implementation of active styling changes for the same menu. Great work!

**Pull request merged**